arraycontext/impl/pyopencl/__init__.py

```diff
@@ -211,6 +212,11 @@ def transform_loopy_program(self, t_unit):
         # accommodate loopy with and without kernel callables
 
         import loopy as lp
+
+        for arg in t_unit.default_entrypoint.args:
+            if isinstance(arg.tags, ParameterValue):
+                t_unit = lp.fix_parameters(t_unit, **{arg.name: arg.tags.value})
+
```

Comment:
PyOpenCLArrayContext.transform_loopy_program is planned for deprecation, maybe this should be a part of subclassed arraycontext to account for these tags.?